lib/net/networking_engine.js

```diff
@@ -530,10 +534,15 @@ shaka.net.NetworkingEngine = class extends shaka.util.FakeEventTarget {
           this.onHeadersReceived_(headers, request, type);
         }
         headersReceivedCalled = true;
+        if (request.requestStartTime) {
```

Comment:
When is this ever unset or zero?  If you're unsure about the timing of when this is invoked, just move `request.requestStartTime = ...` above the call to the plugin.

Reply:
It was more of a case that the linter knows this value can be undefined. I will move the assignment as you suggest and an a type cast as we know it must be assigned at this point.